multichain-testing/README.md

```diff
@@ -0,0 +1,62 @@
+# Starship 
+
+End-to-end testing environment for fully simulated chains, powered by [Starship](https://docs.cosmology.zone/starship).
+
+
+## Configuration
+
+The current commands will read from [`config.yaml`](./config.yaml) to build a multi-chain teting environment. Currently, the image includes `agoric`, `osmosis`, and `cosmos-hub` chains and a hermes relayer between each.
+
+
+## Initial Setup
+
+Ensure you have `kubectl`, `kind`, `helm`, and `yq` installed on your machine. For convenience, the following command will install dependencies:
+
+```sh
+make setup-deps
+```
+
+You will need a `kind` cluster:
+
+```sh
+make setup-kind
+```
+
+## Getting Started
+
+```sh
+# install helm chart and start starship service
+make install
+
+# NOTE: it takes about 10-12 minutes for the above to finish setting up. Use `watch kubectl get pods` to confirm all pods are up and running before running the next command.
+
+# expose ports on your local machine. useful for testing dapps
+make port-forward
+
+# stop the containers and port-forwarding
+make stop
+```
+
+To setup finish setting up Agoric, also run:
+
+```bash
+make fund-provision-poool
+```
+
```

Comment:
Following this guide everything works except the two steps below. Would be helpful to provide an example:

```suggestion
For the steps below, you must import a key to `agd` or create a new one. For example, we can generate one we can use as `ADDR`:

`kubectl exec -i agoriclocal-genesis-0 -c validator -- agd keys add user1`

```

Reply:
Thanks, added this note as a note